datafusion/core/src/datasource/avro_to_arrow/schema.rs

```diff
@@ -144,14 +148,17 @@ fn schema_to_field_with_props(
         AvroSchema::Decimal(DecimalSchema {
             precision, scale, ..
         }) => DataType::Decimal128(*precision as u8, *scale as i8),
+        AvroSchema::BigDecimal => DataType::LargeBinary,
```

Comment:
I wondered if there would be a more useful mapping for `BigDecimal`, but it seems like binary is good for now:
https://docs.rs/apache-avro/latest/apache_avro/schema/enum.Schema.html#variant.BigDecimal